src/Symfony/Bridge/PhpUnit/bin/simple-phpunit

```diff
@@ -15,8 +15,17 @@
 
 error_reporting(-1);
 
-// PHPUnit 4.8 does not support PHP 7, while 5.1 requires PHP 5.6+
-$PHPUNIT_VERSION = PHP_VERSION_ID >= 50600 ? getenv('SYMFONY_PHPUNIT_VERSION') ?: '5.7' : '4.8';
+if (PHP_VERSION_ID >= 70200) {
+    // PHPUnit 6 is required for PHP 7.2+
+    $PHPUNIT_VERSION = getenv('SYMFONY_PHPUNIT_VERSION') ?: '6.3';
+} elseif (PHP_VERSION_ID >= 50600) {
+    // PHPUnit 4 does not support PHP 7
+    $PHPUNIT_VERSION = getenv('SYMFONY_PHPUNIT_VERSION') ?: '5.7';
+} else {
+    // PHPUnit 5.1 requires PHP 5.6+
+    $PHPUNIT_VERSION = getenv('SYMFONY_PHPUNIT_VERSION') ?: '4.8';
```

Comment:
This changes the existing behavior. For PHP < 5.6, the env variable was ignored before, forcing to use 4.8, as this is the only usable version there:

- newer versions are not supported on these PHP runtimes
- older versions than 4.8 are not supported by the bridge.

I would keep this behavior (anyway, changing it is a BC break).